### PR TITLE
[4365] set name to lowercase and then capitalize with css

### LINKED ISSUE #4365: One of letters is written as capital letter in the middle of word in breadcrumbs in my account and compare list
**Description**:
One of letters is written as capital letter in the middle of word in breadcrumbs in my account and compare list

**Expected behavior**:
Breadcrumbs are written:
Compare: Home -> Product Compare
My Account: Home -> My Account

**To Reproduce**:
1. Open FE
2. Log in
3. Open Compare list
4. Open My Account 

**Actual result**:
One of letters is written as capital letter in breadcrumbs in my account and compare list
Compare: Home -> Pro**D**uct Compare
My Account: Home -> My Accou**N**t
Screenshot 2022-03-01 at 14 35 46
Screenshot 2022-03-01 at 14 36 06

### PR DESCRIPTION
**Pull request by pablobentivengo**
**Related issue(s):**
* Fixes #4365

**Problem:**
* One of letters is written as capital letter in the middle of word in breadcrumbs in my account and compare list

**In this PR:**
* Set name to lowercase and then capitalize with css

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
1. Breadcrumbs aren't written with capital letters on search result page if user entered word with Caps lock
Screenshot 2022-03-11 at 15 01 24

File: packages/scandipwa/src/component/Breadcrumb/Breadcrumb.component.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ export class Breadcrumb extends PureComponent {
         } = this.props;
 
         const url = this.getLinkUrl() || {};
+        const nameToString = name.toString();
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/Breadcrumb/Breadcrumb.component.js:62]: Name is an object?

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/Breadcrumb/Breadcrumb.component.js (hunk 1 above)
Comment (AleksandrsKondratjevs) [reply packages/scandipwa/src/component/Breadcrumb/Breadcrumb.component.js:62]: String(name)

Comment (pablobentivengo) [reply packages/scandipwa/src/component/Breadcrumb/Breadcrumb.component.js]: Done

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**